### PR TITLE
All claims transform fixes

### PR DESCRIPTION
**Pull request by devin-mccurdy**
## Description
Adds a null check for `ratedDisabilities` and PTSD behavior changes.

Related to https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/16098

## Testing done

- [x] Can correctly submit all claims flow without a rated disability .
- [x] Can submit all claims flow without PTSD behavior changes.

## Screenshots
N/A

## Acceptance criteria
- [x] Can submit all claims flow without errors

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by aaronjlech — APPROVED** [review]

**Comment by dmccurdybah** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Could I get a code review on this at your earliest convenience? Thanks!

File: src/applications/disability-benefits/all-claims/utils.jsx [hunk 1]
```diff
@@ -303,7 +303,7 @@ export function getFlatIncidentKeys() {
   return incidentKeys;
 }
 
-export function getPtsdChangeText(changeFields) {
+export function getPtsdChangeText(changeFields = {}) {
```
Comment (jbalboni) [inline src/applications/disability-benefits/all-claims/utils.jsx]: Are there unit tests for this function? Might be worth adding one for this case.

Comment (dmccurdybah) [reply src/applications/disability-benefits/all-claims/utils.jsx]: Yes, good idea.

Comment (dmccurdybah) [reply src/applications/disability-benefits/all-claims/utils.jsx]: @jbalboni - I added the test you asked for, could you take a look at this when you get a chance?

**Review by jbalboni — APPROVED** [review]

**Pull request merged**